examples/custom-interactions/webpack.config.js

```diff
@@ -2,7 +2,7 @@
 const {resolve} = require('path');
```

Comment:
Mapbox `v0.38.0` enforces Node `v6.4` and higher. We're gonna start to be clear on the API here. I still need to update the README as part of this PR.